src/cmd/build.rs

```diff
@@ -436,7 +436,6 @@ fn do_optimization(
             We use this tool to optimize the size of your contract's Wasm binary.\n\n\
             wasm-opt is part of the binaryen package. You can find detailed\n\
             installation instructions on https://github.com/WebAssembly/binaryen#tools.\n\n\
-
```

Comment:
Thanks, that warning was driving me nuts, but making a PR for it alone would have been overkill.

Reply:
Small PRs are fine IMO